packages/plugins/model-define/tensorflow-resnet-model-define/src/index.ts

```diff
@@ -122,35 +107,28 @@ const resnetModelDefine: ModelDefineType = async (data: ImageDataset, args: Mode
     model.load_weights(path.join(recoverPath, 'weights.h5'));
   }
 
-  model.compile(
-    boa.kwargs({
-      optimizer: Adam(
-        boa.kwargs({
-          lr: learningRate,
-          decay
-        })
-      ),
-      loss: loss,
-      metrics: metrics
-    })
-  );
+  model.compile(boa.kwargs({
+    optimizer: Adam(boa.kwargs({
+      lr: learningRate,
+      decay
+    })),
+    loss: loss,
+    metrics: metrics
+  }));
 
   const result: UniModel = {
     model,
     metrics: metrics,
-    predict: async function(inputData: ImageSample) {
+    predict: async function (inputData: ImageSample) {
       let image = tf.io.read_file(inputData.data);
-      image = tf.image.decode_jpeg(
-        image,
-        boa.kwargs({
-          channels: 3
-        })
-      );
+      image = tf.image.decode_jpeg(image, boa.kwargs({
+        channels: 3
+      }));
       const shape = tf.shape(image).numpy();
-      return this.model.predict(tf.reshape(image, [1].concat(...shape.slice(0, 3)))).toString();
+      return this.model.predict(tf.reshape(image, [ 1 ].concat(...shape.slice(0, 3)))).toString();
     }
   };
   return result;
 };
 
-export default resnetModelDefine;
+export default resnetModelDefine;
```

Comment:
We might need an EOL here?

```suggestion
export default resnetModelDefine;

```